src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -645,6 +645,8 @@ const char* Hole::ThreadDirectionEnums[]  = { "Right", "Left", NULL};
 
 PROPERTY_SOURCE(PartDesign::Hole, PartDesign::ProfileBased)
 
+const App::PropertyAngle::Constraints floatAngle = { FLT_MIN, 360, 1.0 };

```

Comment:
The same applies here as in https://github.com/FreeCAD/FreeCAD/pull/4553

Reply:
Thanks, now fixed.